pkg/scheduler/framework/job_updater.go

```diff
@@ -61,9 +58,6 @@ func isPodGroupConditionsUpdated(newCondition, oldCondition []scheduling.PodGrou
 
 		newTime := newCond.LastTransitionTime
 		oldTime := oldCond.LastTransitionTime
-		if TimeJitterAfter(newTime.Time, oldTime.Time, jobConditionUpdateTime, jobConditionUpdateTimeJitter) {
```

Comment:
What's the backgroud of update podgroup in a period of time?

Reply:
This item is added to reduce the number of access times of the API server. However, this item does not take effect.

If the conditions are inconsistent and the duration exceeds one minute, update the API server.

I have verified that the pod group status does not change when the pod group is in the running state. When the pod group is in the pending or inqueue state, the pod group status does not change frequently. Therefore, I think the 1-minute interval can be deleted.